README.MD

```diff
@@ -12,4 +12,50 @@ This mod doesn't do anything by itself! You have to create a custom item [like t
+The prefab should be set up so that it's front is along the **-Z** axis and the top along the **+Y** axis.  The prefab should include a box collider or mesh collider to enable interaction.  Box collider is preferred unless it's excessively sized relative to the actual mesh.
+
+### Shelf Prefab
+
+The shelf prefab is an optional prefab.  If provided, this is what will be placed on the shelf.  It can include multiple copies of your item, or a creative arrangement.  It should be oriented with the front along the **+Z** axis and the top along the **+Y** axis. It should have a single box collider that wraps the entire visible portion of the prefab.
+
+### Icons
+
+Icons are simply png files (keep them reasonably sized, they only show up around 300x300 pixels, maybe a bit bigger on 4k screens...).  In Unity they must be marked as sprites (after importing the png, select it and in the properties window on the right change from texture to sprite).
+
+### Shelf Size
+
+The shelf size is determined in one of two ways in order of priority:
+
+1. ShelfBounds json value - this is an XYZ Vector3.  Specifically X determines the width of the shelf space needed, and Y determines the height needed.  Z is not currently used.
+1. ShelfPrefab box collider size
+
+If the size cannot be determined in one of those ways, a default shelf space of .35 x .3 meters will be assumed.
+
+### JSON Fields
+
+**Required Fields:**
+
+- Name - string : This is the name that will be displayed in game
+- Description - string: This will be displayed when hovering over the item to purchase it.
+- Amount - integer: This is the number of items available to purchase
+- Price - integer: This is how much the item will cost in career mode (affected by career settings) 
+- AssetBundleName - string: This is the filename of the built asset bundle
+- PrefabPath - string: This is the path within the asset bundle to get to your item prefab (this should be listed in your asset bundle manifest and end in .prefab)
+- IconStandardPath - string: This is the path within the asset bundle to the icon asset for inventory view.
+- IconDroppedPath - string: This is the path within the asset bundle to the icon asset for locked slots in inventory where the item is missing. 
+
+**Optional Fields:**
+- ShelfPrefabPath - string: This is the path within the asset bundle to get to your shelf prefab if provided.  If missing, the item prefab will be used rotated 180 degrees.
+- ShelfBounds - Vector3: This is the width, height, and depth (x, y, z) of the item on the shelf
+- ShelfScale - Vector3: A scaling factor to apply to the item placed on the shelf (either the shelf prefab or the item prefab) to help it fit.  Scaling is per vector.
+- ShelfRotation - Vector3: Rotation in Eurler angles to apply to the shelf item.  If provided, this will override the automatic rotation applied to item prefab if no shelf prefab is provided.

```

Comment:
Eurler -> Euler.

I think "Rotation in degrees" might be better, not everyone knows what a Euler is

Reply:
Updated to specify Roll,Pitch,Yaw and degrees.  This is as opposed to a quaternion rotation.